### DISCUSSION TITLE
Disable automounting

### DISCUSSION THREAD

**Discussion by hektr** [lxqt/lxqt#2017] [Q&A] [answered]
Now it by default automatically mounts everything. Why don't mount by pressing on device button on pcmanfm? It takes a couple of seconds, but gives us more freedom :)

**Comment by tsujan** [comment]
> Now it by default automatically mounts everything

What's "it"?

**Reply by hektr** [reply]
gvfs.It comes with lxqt in debian. May be, it's idea of debian maintainers. Yes, i think, it's debian and not lxqt

**Reply by tsujan** [reply]
GVFS doesn't auto-mount anything. (Without GVFS, many functionalities would be missing.)

If the two relevant boxes are uncheched in `pcmanfm-qt → Preferences → Volume → Auto Mount`, find which other program does auto-mounting in your case.

**Reply by hektr** [reply]
wow! It's working :) Thanks!

**Reply by stefonarch** [reply]
I moved this to Q&A, but the problem is that you cannot mark as "answer" any posting in those "sub-threads".

**Comment by hektr** [comment, marked as answer]
I just uncheked two relevant boxes in pcmanfm-qt → Preferences → Volume → Auto Mount
